CHANGELOG.rst

```diff
@@ -1,5 +1,5 @@
-xCORE-200 DSP library change log
-================================
+DSP library change log
```

Comment:
The warning generated by the change log check is just that it cannot automatically confirm the title is correct, not that is incorrect.

This change will introduce inconsistencies with the title used in the readme, app note, and documentation so I don’t think it is advisable.

Also it’s worth noting that this library will not build for XS1, if that’s what users are looking for.